library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -35,10 +35,7 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```

Comment:
I should send you a gif of me trying to do that, but my IDE keeps reverting to `import java.util.*;`. It's actually pretty funny.

I'll just change IDE settings and amend this PR. :)